commands/run.js

```diff
@@ -34,8 +34,8 @@ module.exports = function (topic) {
 
 let run = async function (context, heroku) {
   if (context.args.length === 0) {
-    cli.error(`Error: Requires one process type\n ${usage} `)
-    process.exit(1)
+    cli.error(`Error: Requires one process type\n ${usage} `, 1)
+    return
```

Comment:
Do you need to return? If the line above ultimately calls `process.exit`, I would assume it isn't necessary.

Reply:
We shouldn't, no: https://devcenter.heroku.com/articles/developing-cli-plugins#warnings-and-errors
But for some reason, the cli doesn't actually exit and the rest of the function is executed here. I intend to ask CLI about it.

ping @heroku/cli